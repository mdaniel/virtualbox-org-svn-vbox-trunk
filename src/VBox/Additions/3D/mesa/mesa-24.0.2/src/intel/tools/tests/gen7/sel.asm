(+f0.0) sel(8)  g47<1>UD        g12<4>UD        g13<4>UD        { align16 1Q };
(-f0.0) sel(8)  g25<1>.xyUD     g13<4>.zwwwUD   0x40000000UD    { align16 1Q };
(+f0.0.any4h) sel(8) g30<1>UD   g13<4>UD        g12<4>UD        { align16 1Q };
(+f0.0.all4h) sel(8) g16<1>UD   g8<4>UD         g9<4>UD         { align16 1Q };
(+f0.0) sel(8)  g2<1>UD         g31<8,8,1>UD    g34<8,8,1>UD    { align1 1Q };
(+f0.0) sel(8)  g124<1>UD       g67<8,8,1>UD    0x3f800000UD    { align1 1Q };
(+f0.0) sel(16) g2<1>UD         g35<8,8,1>UD    g41<8,8,1>UD    { align1 1H };
(+f0.0) sel(16) g120<1>UD       g27<8,8,1>UD    0x3f800000UD    { align1 1H };
sel.ge(8)       g64<1>F         g9<8,8,1>F      0x0F  /* 0F */  { align1 1Q };
(-f0.0) sel(8)  g16<1>UD        g20<8,8,1>UD    0x00000000UD    { align1 1Q };
sel.ge(16)      g17<1>F         g3<8,8,1>F      0x0F  /* 0F */  { align1 1H };
(-f0.0) sel(16) g23<1>UD        g21<8,8,1>UD    0x00000000UD    { align1 1H };
sel.l(8)        g13<1>.xyzD     g6<0>.xyzzD     g5.4<0>.zD      { align16 1Q };
sel.ge(8)       g3<1>.yF        g7<4>.xF        0x0F  /* 0F */  { align16 1Q };
sel.l(8)        g11<1>.xF       g7<4>.wF        0x43000000F  /* 128F */ { align16 1Q };
(-f0.0.z) sel(8) g3<1>.zUD      g14<4>.xUD      0x00000000UD    { align16 1Q };
sel.l(8)        g14<1>UD        g6<0>UD         g6.4<0>UD       { align16 1Q };
(+f0.0.x) sel(8) g32<1>.xUD     g12<4>.yUD      0x41a80000UD    { align16 1Q };
(-f0.0.x) sel(8) g33<1>.xUD     g32<4>.xUD      0x41b80000UD    { align16 1Q };
sel.ge(8)       g2<1>F          (abs)g7<8,8,1>F (abs)g8<8,8,1>F { align1 1Q };
sel.ge(16)      g2<1>F          (abs)g10<8,8,1>F (abs)g12<8,8,1>F { align1 1H };
(+f0.0.x) sel(8) g25<1>.xUD     g23<4>.yUD      g23<4>.xUD      { align16 1Q };
sel.sat.l(8)    g116<1>F        g2<4>F          0x3f000000F  /* 0.5F */ { align16 1Q };
sel.l(8)        g13<1>.xF       g1<0>.wF        g1<0>.zF        { align16 1Q };
sel.ge(8)       g13<1>.xF       g1<0>.wF        g1<0>.zF        { align16 1Q };
(-f0.0.any4h) sel(8) g67<1>.xUD g63<4>.xUD      0x00000000UD    { align16 1Q };
(+f0.0.x) sel(8) g17<1>.xF      g5.4<0>.zF      -g5.4<0>.zF     { align16 1Q };
sel.l(8)        g124<1>F        g2.3<0,1,0>F    g2.2<0,1,0>F    { align1 1Q };
sel.l(16)       g120<1>F        g2.3<0,1,0>F    g2.2<0,1,0>F    { align1 1H };
sel.ge(8)       g13<1>.xyzD     g6<0>.xyzzD     g5.4<0>.zD      { align16 1Q };
sel.ge(8)       g13<1>.xyUD     g5.4<0>.zwwwUD  g6<0>.xyyyUD    { align16 1Q };
(+f0.0.any4h) sel(8) g17<1>.xUD g8<4>.xUD       0x00000001UD    { align16 1Q };
sel.ge(8)       g12<1>.xD       g5.4<0>.zD      -1D             { align16 1Q };
sel.l(8)        g14<1>.xD       g12<4>.xD       1D              { align16 1Q };
sel.l(8)        g3<1>UD         g2.1<0,1,0>UD   0x00000001UD    { align1 1Q };
sel.l(16)       g3<1>UD         g2.1<0,1,0>UD   0x00000001UD    { align1 1H };
sel.ge(8)       g4<1>D          g3<0,1,0>D      -252D           { align1 1Q };
sel.l(8)        g5<1>D          g4<8,8,1>D      254D            { align1 1Q };
sel.ge(16)      g4<1>D          g3<0,1,0>D      -252D           { align1 1H };
sel.l(16)       g6<1>D          g4<8,8,1>D      254D            { align1 1H };
sel.sat.l(8)    g116<1>F        g1<0>F          g3<4>F          { align16 1Q };
sel.l(8)        g6<1>F          g3<8,8,1>F      0x40400000F  /* 3F */ { align1 1Q };
sel.l(16)       g2<1>F          g20<8,8,1>F     0x40400000F  /* 3F */ { align1 1H };
(+f0.0) sel(8)  g28<1>.xyF      (abs)g6<0>.xyyyF g5.4<0>.zwwwF  { align16 1Q };
(+f0.0) sel(8)  g31<1>.xyUD     g10<4>.xyyyUD   0x3f800000UD    { align16 1Q };
(-f0.0) sel(8)  g38<1>.xyF      (abs)g35<4>.xyyyF 0x3f800000F  /* 1F */ { align16 1Q };
sel.l(8)        g13<1>.xUD      g11<4>.xUD      0x00000007UD    { align16 1Q };
(+f1.0) sel(4)  g17<1>.xUD      g15.4<4>.xUD    g15<4>.xUD      { align16 WE_all 1N };
(+f0.0) sel(8)  g57<1>F         (abs)g2.3<0,1,0>F g2<0,1,0>F    { align1 1Q };
(-f0.0) sel(8)  g29<1>F         (abs)g26<8,8,1>F 0x3f800000F  /* 1F */ { align1 1Q };
(+f0.0) sel(16) g8<1>F          (abs)g2.3<0,1,0>F g2<0,1,0>F    { align1 1H };
(-f0.0) sel(16) g55<1>F         (abs)g14<8,8,1>F 0x3f800000F  /* 1F */ { align1 1H };
(-f0.0.x) sel(8) g32<1>.xF      (abs)g29<4>.xF  0x3f800000F  /* 1F */ { align16 1Q };
sel.sat.ge(8)   g116<1>F        g25<4>F         0xbf800000F  /* -1F */ { align16 1Q };
sel.sat.l(8)    g46<1>F         g45<8,8,1>F     0x3f000000F  /* 0.5F */ { align1 1Q };
sel.sat.l(16)   g8<1>F          g83<8,8,1>F     0x3f000000F  /* 0.5F */ { align1 1H };
