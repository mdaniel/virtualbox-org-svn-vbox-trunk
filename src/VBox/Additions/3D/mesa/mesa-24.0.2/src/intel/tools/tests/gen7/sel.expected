02 01 61 00 21 04 ef 25 84 01 6e 00 a4 01 6e 00
02 01 71 00 21 0c 23 23 ae 01 6f 00 00 00 00 40
02 01 66 00 21 04 cf 23 a4 01 6e 00 84 01 6e 00
02 01 67 00 21 04 0f 22 04 01 6e 00 24 01 6e 00
02 00 61 00 21 04 40 20 e0 03 8d 00 40 04 8d 00
02 00 61 00 21 0c 80 2f 60 08 8d 00 00 00 80 3f
02 00 81 00 21 04 40 20 60 04 8d 00 20 05 8d 00
02 00 81 00 21 0c 00 2f 60 03 8d 00 00 00 80 3f
02 00 60 04 bd 7f 00 28 20 01 8d 00 00 00 00 00
02 00 71 00 21 0c 00 22 80 02 8d 00 00 00 00 00
02 00 80 04 bd 7f 20 22 60 00 8d 00 00 00 00 00
02 00 91 00 21 0c e0 22 a0 02 8d 00 00 00 00 00
02 01 60 05 a5 14 a7 21 c4 00 0a 00 ba 00 0a 00
02 01 60 04 bd 7f 62 20 e0 00 60 00 00 00 00 00
02 01 60 05 bd 7f 61 21 ef 00 6f 00 00 00 00 43
02 01 74 00 21 0c 64 20 c0 01 60 00 00 00 00 00
02 01 60 05 21 04 cf 21 c4 00 0e 00 d4 00 0e 00
02 01 62 00 21 0c 01 24 85 01 65 00 00 00 a8 41
02 01 72 00 21 0c 21 24 00 04 60 00 00 00 b8 41
02 00 60 04 bd 77 40 20 e0 20 8d 00 00 21 8d 00
02 00 80 04 bd 77 40 20 40 21 8d 00 80 21 8d 00
02 01 62 00 21 04 21 23 e5 02 65 00 e0 02 60 00
02 01 60 85 bd 7f 8f 2e 44 00 6e 00 00 00 00 3f
02 01 60 05 bd 77 a1 21 2f 00 0f 00 2a 00 0a 00
02 01 60 04 bd 77 a1 21 2f 00 0f 00 2a 00 0a 00
02 01 76 00 21 0c 61 28 e0 07 60 00 00 00 00 00
02 01 62 00 bd 77 21 22 ba 00 0a 00 ba 40 0a 00
02 00 60 05 bd 77 80 2f 4c 00 00 00 48 00 00 00
02 00 80 05 bd 77 00 2f 4c 00 00 00 48 00 00 00
02 01 60 04 a5 14 a7 21 c4 00 0a 00 ba 00 0a 00
02 01 60 04 21 04 a3 21 be 00 0f 00 c4 00 05 00
02 01 66 00 21 0c 21 22 00 01 60 00 01 00 00 00
02 01 60 04 a5 1c 81 21 ba 00 0a 00 ff ff ff ff
02 01 60 05 a5 1c c1 21 80 01 60 00 01 00 00 00
02 00 60 05 21 0c 60 20 44 00 00 00 01 00 00 00
02 00 80 05 21 0c 60 20 44 00 00 00 01 00 00 00
02 00 60 04 a5 1c 80 20 60 00 00 00 04 ff ff ff
02 00 60 05 a5 1c a0 20 80 00 8d 00 fe 00 00 00
02 00 80 04 a5 1c 80 20 60 00 00 00 04 ff ff ff
02 00 80 05 a5 1c c0 20 80 00 8d 00 fe 00 00 00
02 01 60 85 bd 77 8f 2e 24 00 0e 00 64 00 6e 00
02 00 60 05 bd 7f c0 20 60 00 8d 00 00 00 40 40
02 00 80 05 bd 7f 40 20 80 02 8d 00 00 00 40 40
02 01 61 00 bd 77 83 23 c4 20 05 00 be 00 0f 00
02 01 61 00 21 0c e3 23 44 01 65 00 00 00 80 3f
02 01 71 00 bd 7f c3 24 64 24 65 00 00 00 80 3f
02 01 60 05 21 0c a1 21 60 01 60 00 07 00 00 00
02 03 41 00 21 04 21 22 f0 01 60 04 e0 01 60 00
02 00 61 00 bd 77 20 27 4c 20 00 00 40 00 00 00
02 00 71 00 bd 7f a0 23 40 23 8d 00 00 00 80 3f
02 00 81 00 bd 77 00 21 4c 20 00 00 40 00 00 00
02 00 91 00 bd 7f e0 26 c0 21 8d 00 00 00 80 3f
02 01 72 00 bd 7f 01 24 a0 23 60 00 00 00 80 3f
02 01 60 84 bd 7f 8f 2e 24 03 6e 00 00 00 80 bf
02 00 60 85 bd 7f c0 25 a0 05 8d 00 00 00 00 3f
02 00 80 85 bd 7f 00 21 60 0a 8d 00 00 00 00 3f
